ownCloud/Settings/SecuritySettingsSection.swift

```diff
@@ -22,244 +22,283 @@ import ownCloudSDK
 
 // MARK: - SecurityAskfrequency
 @objc enum SecurityAskFrequency: Int {
-    case always = 0
-    case oneMinute = 60
-    case fiveMinutes = 300
-    case thirtyMinutes = 1800
-
-    static let all = [always, oneMinute, fiveMinutes, thirtyMinutes]
-
-    func toString() -> String {
-        switch self {
-        case .always:
-            return "Always".localized
-        case .oneMinute:
-            return "After 1 minute".localized
-        case .fiveMinutes:
-            return "After 5 minutes".localized
-        case .thirtyMinutes:
-            return "After 30 minutes".localized
-        }
-    }
+	case always = 0
+	case oneMinute = 60
+	case fiveMinutes = 300
+	case thirtyMinutes = 1800
+
+	static let all = [always, oneMinute, fiveMinutes, thirtyMinutes]
```

Comment:
Maybe not on the scope of this PR but with Swift 4.2 an enum could be `CaseIterable` and this all workaround could be deleted. If we don't address this here, we should open an issue or left a `//TODO` so the linter can warn us next time.

Reply:
Good point. The next commit will take advantage of `CaseIterable`.